pyproject.toml

```diff
@@ -88,4 +88,5 @@ ignore-words-list = [
 	"textin",
 	"thirdparty",
 	"vai",
+	"streamin",
```

Comment:
Is this added intentionally?

Reply:
The list is ordered alphabetically so this should go just before `"te"`.